### PR TITLE
[GS End User][W-8545403] Checklist section 3

### PR DESCRIPTION
**Pull request by fparodi-salesforce**
# Critical Changes

# Changes
Create custom metadata to add new section 3 and his sub section in the End User GS page
# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom Metadata
   - GetStartedChecklistSection.Track_Donations
   - GetStartedChecklistItem.Donation_Organization
   - GetStartedChecklistItem.Donation_Individual
   - GetStartedChecklistItem.Collect_Gifts
   - GetStartedChecklistItem.Capture_Donor_Intent
   - GetStartedChecklistItem.Track_Donation
<br/>

- Custom Label
   - gseuChecklistItemCaptureDonorIntentDesc
   - gseuChecklistItemCaptureDonorIntentLinkLabel
   - gseuChecklistItemCaptureDonorIntentPriBtnLabel
   - gseuChecklistItemCaptureDonorIntentTitle
   - gseuChecklistItemCollectGiftsDesc
   - gseuChecklistItemCollectGiftsLinkLabel
   - gseuChecklistItemCollectGiftsPriBtnLabel
   - gseuChecklistItemCollectGiftsTitle
   - gseuChecklistItemDonationIndividualDesc
   - gseuChecklistItemDonationIndividualLinkLabel
   - gseuChecklistItemDonationIndividualPriBtnLabel
   - gseuChecklistItemDonationIndividualTitle
   - gseuChecklistItemDonationOrganizationDesc
   - gseuChecklistItemDonationOrganizationLinkLabel
   - gseuChecklistItemDonationOrganizationPriBtnLabel
   - gseuChecklistItemDonationOrganizationTitle
   - gseuChecklistItemTrackDonationDesc
   - gseuChecklistItemTrackDonationLinkLabel
   - gseuChecklistItemTrackDonationPriBtnLabel
   - gseuChecklistItemTrackDonationTitle
   - gseuChecklistTrackDonationDesc
   - gseuChecklistTrackDonationTitle
# Deleted Metadata

### CODE REVIEW

**Comment by fparodi-salesforce** [comment]
[W-8545403 ](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008kxN0IAI/view)

**Review by force2b — CHANGES_REQUESTED** [review]
This has the same two issues that the previous PR I just reviewed has:

- A `gseuAssets` folder was created in #6150 and should be used here (or deleted in #6150).
- The use of an apostrophe in custom labels needs to be replaced with `&apos;`. Be sure to check for double quotes as well and other characters that may need escaping.

**Comment by fparodi-salesforce** [comment]
@force2b The problem to use another is the checklist Item component try to find the image in the gsAssets/step folder,
to use another folder hay need past list filter to change the find image folder.
You prefer I update the checklist component to can select different asset folder?

**Comment by force2b** [comment]
The `gseuAssets` folder was created by another PR to put all assets related to the End User page. If that will not work then it should be removed from the other PR. My specific issue is that we can't put some End User resources in one folder and others in another folder. It must be consistent - whichever way your team decides to go. @dcaballeroSf

**Comment by force2b** [comment]
@dcaballeroSf What is the decision regarding the `gseuAssets` folder? I noticed that the 2 PR's assigned to me today use `gsAssets`, while #6150 still uses `gseuAssets`.

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist item FundraisingProcess in Get Started</shortDescription>
         <value>Visualize Your Fundraising Processes</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7336]: I recommend changing this to: Description of Capture Donor Intent checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>When you raise funds for a specific purpose, you want to be sure you are honoring the donor intent. Ensure donations are allocated correctly with General Allocation Units.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Capture Donor Intent in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7344]: I recommend changing this to: Link label of Capture Donor Intent checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Donation Allocations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Capture Donor Intent in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7352]: I recommend changing this to: Primary button label of Capture Donor Intent checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 4]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Create Allocations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7360]: I recommend changing this to: Description of Capture Donor Intent checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 5]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Capture Donor Intent with Donation Allocations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Collect Gifts in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7368]: I recommend changing this to: Description of Collect Gifts checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 6]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Monthly giving can provide a predictable source of nonprofit revenue, and sustaining donors are often a nonprofit&apos;s most loyal and committed supporters. Track monthly giving by creating a Recurring Donation.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Collect Gifts in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7376]: I recommend changing this to: Link label of Collect Gifts checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 7]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Recurring Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Collect Gifts in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7384]: I recommend changing this to: Primary button label of Collect Gifts checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 8]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Create Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Collect Gifts in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7392]: I think should be: Title of Collect Gifts checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 9]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Collect Sustaining Gifts with Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Individual in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7400]: I recommend changing this to: Description of Individual Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 10]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>For most nonprofits, donations come from individual contributions. Track individual donations by creating an Opportunity from a Contact record.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Donation Individual in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7408]: I recommend changing this to: Link label of Individual Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 11]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Individual Donations in Trailhead</value>
+    </labels>
+     <labels>
+    <fullName>gseuChecklistItemDonationIndividualPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Individual in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7416]: I recommend changing this to: Primary button label of Individual Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 12]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Enter Individual Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Individual in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7424]: I recommend changing this to: Title of Individual Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 13]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Enter a Donation from an Individual</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7432]: I recommend changing this to: Description of Organization Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 14]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Institutional and corporate giving are an important source of nonprofit support. Track organizational giving by creating an Opportunity from an Account record.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Donation Organization in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7440]: I recommend changing this to: Link label of Organization Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 15]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Organization Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Organization in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7448]: I recommend changing this to: Primary button label of Organization Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 16]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Enter Organization Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7456]: I think this should be: Title of Organization Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 17]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Enter a Donation from an Organization</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Track Donation in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7464]: I recommend changing this to: Description of Track Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 18]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Campaigns let you to track your constituent outreach in Salesforce. Use the Primary Campaign field on Opportunities to better evaluate your fundraising efforts.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Track Donation in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7472]: I recommend changing this to: Link label of Track Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 19]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Learn more about Campaigns in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Track Donation in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7480]: I recommend changing this to: Primary button label of Track Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 20]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Track Constituent Outreach</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Track Donation in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7488]: I think this should be: Title of Track Donation checklist item in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 21]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>Track Donation Sources with Campaigns</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Track Donation in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7496]: I recommend changing this to: Description of Track Donation checklist in Get Started for End Users.

File: src/labels/CustomLabels.labels [hunk 22]
```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
...
+        <value>In NPSP, you track donations, grants, pledges, and other types of revenue using the Opportunity object. You can create donations from an individual or from an organization such as a foundation, business, or government entity.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistTrackDonationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist Track Donation in GS for End User</shortDescription>
```
Comment (smarovich) [inline src/labels/CustomLabels.labels:7504]: I recommend changing this to: Title of Track Donation checklist in Get Started for End Users.

File: src/labels/CustomLabels.labels (hunk 15 above)
Comment (fparodi-salesforce) [reply src/labels/CustomLabels.labels:7448]: @smarovich the max length for a short description is 80 and this message has 89

File: src/labels/CustomLabels.labels (hunk 19 above)
Comment (fparodi-salesforce) [reply src/labels/CustomLabels.labels:7480]: @smarovich the max length for a short description is 80 and this message has 82

File: src/labels/CustomLabels.labels (hunk 11 above)
Comment (fparodi-salesforce) [reply src/labels/CustomLabels.labels:7416]: @smarovich the max length for a short description is 80 and this message has 87

File: src/labels/CustomLabels.labels (hunk 15 above)
Comment (smarovich) [reply src/labels/CustomLabels.labels:7448]: Ok, let's use: Primary button label of Org Donation checklist item in Get Started for End Users

File: src/labels/CustomLabels.labels (hunk 19 above)
Comment (smarovich) [reply src/labels/CustomLabels.labels:7480]: OK, let's use: Primary button label of Track Donation checklist item in GetStarted for End User

File: src/labels/CustomLabels.labels (hunk 11 above)
Comment (smarovich) [reply src/labels/CustomLabels.labels:7416]: Do we need the word primary in the description? Could we just use: Button label of Individual Donation checklist item in Get Started for End Users.

**Review by force2b — APPROVED** [review]

**Review by smarovich — APPROVED** [review]
Labels look great!

**Pull request merged**